### PR TITLE
fix(controls): update region annotation icon

### PR DESCRIPTION
**Pull request by mickr**
Updating to the newest region annotation icon. I increased the size a little bit because the new icon looked a bit small on the toolbar.

### CODE REVIEW

**Comment by jstoffan** [comment]
Do we have a separate change for normalizing the padding of the toggle button with the thumbnail sidebar toggle button?

File: src/lib/AnnotationControls.scss [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-        width: $controls-button-width;
-        height: $controls-button-width;
-        border-radius: 4px;
-
         svg {
             fill: $white;
         }
 
         &.is-active {
+            width: 34px;
```
Comment (jstoffan) [inline src/lib/AnnotationControls.scss:11]: Seems like these measurements should remain the same whether `is-active` is applied or not.

File: src/lib/AnnotationControls.scss [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+            height: 32px;
+            margin: 7px;
```
Comment (jstoffan) [inline src/lib/AnnotationControls.scss:13]: Seems kind of magic number-y.

**Comment by mickr** [comment]
Default State
Screen Shot 2020-05-18 at 4 58 42 PM

Active State
Screen Shot 2020-05-18 at 4 58 51 PM

Focused State
Screen Shot 2020-05-18 at 4 59 01 PM

File: src/lib/AnnotationControls.scss [hunk 3]
```diff
@@ -1,23 +1,32 @@
...
             svg {
                 fill: $black;
             }
         }
+
+        &.is-active::before {
+            position: absolute;
+            z-index: -1;
+            width: 34px;
+            height: 32px;
```
Comment (mxiao6) [inline src/lib/AnnotationControls.scss:25]: Can we use `$controls-button-width`?

File: src/lib/AnnotationControls.scss [hunk 4]
```diff
@@ -1,23 +1,32 @@
...
+            background-color: $white;
+            border-radius: 4px;
```
Comment (mxiao6) [inline src/lib/AnnotationControls.scss:27]: `bdl-border-radius-size`?

File: src/lib/AnnotationControls.ts [hunk 5]
```diff
@@ -153,7 +153,7 @@ export default class AnnotationControls {
         const regionButton = this.controls.add(
             __('region_comment'),
             this.handleClick(onRegionClick, AnnotationMode.REGION),
-            `${CLASS_BOX_CONTROLS_GROUP_BUTTON} ${CLASS_REGION_BUTTON}`,
+            `${CONTROLS_BUTTON_CLASS} ${CLASS_REGION_BUTTON}`,
```
Comment (mxiao6) [inline src/lib/AnnotationControls.ts:156]: `CONTROLS_BUTTON_CLASS` is added in `Controls.js`, right? I think in the future, there will be more annotations buttons. Do we want to remove the group class?

File: src/lib/AnnotationControls.scss (hunk 3 above)
Comment (mickr) [reply src/lib/AnnotationControls.scss:25]: The design calls for these dimensions... not 32x32

File: src/lib/AnnotationControls.ts (hunk 5 above)
Comment (mickr) [reply src/lib/AnnotationControls.ts:156]: I don't think the group class is what we need with the individual buttons.

Comment (mxiao6) [reply src/lib/AnnotationControls.ts:156]: Is `CONTROLS_BUTTON_CLASS` needed? I believe `this.controls` already has it.

**Pull request merged**